### PR TITLE
[slack] Slack directory with basic bot and .gitignore update for .env

### PR DESCRIPTION
**Pull request by Harshg999**

### CODE REVIEW

File: tools/slack/bot.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from slack_sdk import WebClient
+from pathlib import Path
+from dotenv import load_dotenv
+ 
+ #Create .env file and load it
+load_dotenv()
+ 
+client = WebClient(token = os.getenv('SLACK_BOT_TOKEN')) # Add SLACK_BOT_TOKEN in .env 
+ 
+try:
```
Comment (romainr) [inline tools/slack/bot.py:11]: Nice!

Next should we have the bot monitor the channel and reply 'Hello X' when X says 'Hello Hue'?

Bit similar to https://www.digitalocean.com/community/tutorials/how-to-build-a-slackbot-in-python-on-ubuntu-20-04#step-1-%E2%80%94-creating-the-slackbot-in-the-slack-ui

Then we can see how to refactor a bit to clean-up

Comment (Harshg999) [reply tools/slack/bot.py:11]: Sure! For the bot to monitor the slack events and respond to them, we need to look into having a web server.

Comment (romainr) [reply tools/slack/bot.py:11]: Yes, quick & dirty now and we chat in person Thursday about the long term solution

Comment (Harshg999) [reply tools/slack/bot.py:11]: Sounds good

Comment (Harshg999) [reply tools/slack/bot.py:11]: Closing this PR and opening a new one with more changes @romainr

Comment (Harshg999) [reply tools/slack/bot.py:11]: Also shall we change the directory name to "slackbot" ?
"slack" seems too general

**Comment by Harshg999** [comment]
Closing this PR and opening a new one with further changes @romainr

**Comment by Harshg999** [comment]
Also shall we change the directory name to "slackbot" ?
"slack" seems too general. What do you say?

**Comment by romainr** [comment]
:+1: 

And no worries about the directory naming, will follow-up on the archi diagram

**Pull request closed without merging**